project_task_scheduling/models/project_task_scheduling_proposal.py

```diff
@@ -0,0 +1,44 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+from odoo.tools.translate import _
```

Comment:
@ernestotejeda IMO Remove here and Add Directly `from odoo import api, fields, models, _`